api/tests/unit/app_analytics/split_testing/test_unit_split_testing_views.py

```diff
@@ -0,0 +1,171 @@
+import json
+from typing import Callable
+
+from app_analytics.split_testing.models import SplitTest
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from environments.identities.models import Identity
+from environments.models import Environment
+from environments.permissions.constants import VIEW_ENVIRONMENT
+from features.models import Feature
+from features.multivariate.models import MultivariateFeatureOption
+from features.value_types import STRING
+from projects.models import Project
+
+
+def test_conversion_event(
+    api_client: APIClient,
+    environment: Environment,
+    identity: Identity,
+) -> None:
+    # Given
+    url = reverse("api-v1:conversion-events-list")
+    api_client.credentials(HTTP_X_ENVIRONMENT_KEY=environment.api_key)
+    data = {"identity_identifier": identity.identifier}
+
+    # When
+    response = api_client.post(
+        url, data=json.dumps(data), content_type="application/json"
+    )
+
+    # Then
+    assert response.status_code == status.HTTP_201_CREATED
+    assert response.data == data
+
+
+def test_conversion_event_unauthorized(
+    api_client: APIClient,
+    environment: Environment,
+    identity: Identity,
+) -> None:
+    # Given
+    url = reverse("api-v1:conversion-events-list")
+    data = {"identity_identifier": identity.identifier}
+
+    # When
+    # No environment key header in use.
+    response = api_client.post(
+        url, data=json.dumps(data), content_type="application/json"
+    )
+
+    # Then
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+def test_split_test_list_forbidden(
+    staff_client: APIClient,
+    environment: Environment,
+) -> None:
+    # Given
+    url = reverse("api-v1:split-tests-list")
+    url = f"{url}?environment_id={environment.id}"
+
+    # When
+    response = staff_client.get(url)
+
+    # Then
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+def test_split_test_list(
+    staff_client: APIClient,
+    with_environment_permissions: Callable[[list[str], int], None],
+    environment: Environment,
+    project: Project,
+) -> None:
+    # Given
+    feature = Feature.objects.create(
+        name="single feature for test",
```

Comment:
nit: although it's not enforced in the ORM, we should create a more realistic feature here by not including spaces (since the FE enforces this).

Reply:
Right! Thank you!